server/configuration.go

```diff
@@ -88,7 +88,7 @@ func (dep *DefaultEntryPoints) String() string {
 func (dep *DefaultEntryPoints) Set(value string) error {
 	entrypoints := strings.Split(value, ",")
 	if len(entrypoints) == 0 {
-		return errors.New("Bad DefaultEntryPoints format: " + value)
+		return fmt.Errorf("Bad DefaultEntryPoints format: %s", value)
```

Comment:
Because of [this](https://github.com/golang/go/wiki/CodeReviewComments#error-strings).